docusaurus/docs/dev-docs/backend-customization/controllers.md

```diff
@@ -179,6 +175,121 @@ export default {
 When a new [content-type](/dev-docs/backend-customization/models#content-types) is created, Strapi builds a generic controller with placeholder code, ready to be customized.
 :::
 
+### Sanitization in controllers
+
+:::warning
+As of Strapi v4.7.0 and greater it's strongly recommended you sanitize your incoming request query and parameters utilizing the new `sanitizeParams` function to prevent leaking of private data.
+:::
+
+#### Sanitization when utilizing controller factories
+
+Within the Strapi factories there are 2 functions exposed that can be used for sanitization:
```

Comment:
Are we excluding input sanitization on purpose here?

Reply:
No I just forgot to add it :facepalm: